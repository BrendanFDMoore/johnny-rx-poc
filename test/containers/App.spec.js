import React from 'react';
import { shallow } from 'enzyme';

import App from 'containers/App';

function setup() {
  const props = {
    children: <div key="1" className="child">Hello</div>,
    location: {}
  };

  return shallow(<App {...props} />);
}

describe('App', () => {
  const wrapper = setup();

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
