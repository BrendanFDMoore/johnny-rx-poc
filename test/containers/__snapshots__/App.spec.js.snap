exports[`App should render properly 1`] = `"<div class=\"child\">Hello</div>"`;
